doc/design/007-cli-payments/README.md

```diff
@@ -0,0 +1,141 @@
+# Payment Interface
+
+This design proposes the list of commands to Indy CLI to handle payments.
+
+## Goals and ideas
+
+* Indy CLI should provide ability to perform the main payments operations:
+  * Creation of payment address
+  * Listing of payment addresses
+  * Getting list of UTXO for payment address
+  * Sending payment transaction
+  * Adding fees to transactions
+  * Getting transactions fees amount
+* Abstraction level should correspond to Indy SDK. For example, don't hide UTXO abstraction. In the future we can add
+  new commands to increase abstractions level.
+
+## New CLI commands
+
+### Create payment address
+
+Create payment address for specific payment method in the wallet
+
+```indy-cli
+indy> payment-address create payment_method=<payment-method> seed=[<seed>]
+```
+
+Returns:
+
+* Success or error message
+
+### List payment addresses
+
+List payment addresses for specific payment method in the wallet
+
+```indy-cli
+indy> payment-address list
```

Comment:
Should we specify a payment method in this command?

Reply:
No, This command returns addresses for all payment methods. Each address in fully qualified form and also table that will be printed will contain payment method column